prime-router/metadata/schemas/AZ/az-covid-19-csv.schema

```diff
@@ -6,120 +6,179 @@ trackingElement: standard.testing_lab_specimen_id
 elements:
   - name: standard.sending_application
     type: TEXT
-    csvField: Sending_Application
+    csvFields: 
+      - name: Sending_Application
 
   - name: standard.sending_application_id
     type: HD
-    csvField: Sending_Application_ID
+    csvFields:
+    - name: Sending_Application_ID
 
   - name: standard.testing_lab_name
-    csvField: Lab_Name
+    csvFields:
+    - name: Lab_Name
 
   - name: standard.testing_lab_clia
-    csvField: Lab_CLIA
+    csvFields:
+    - name: Lab_CLIA
 
   - name: standard.testing_lab_street
-    csvField: Lab_Address
+    csvFields:
+    - name: Lab_Address
 
   - name: standard.testing_lab_city
-    csvField: Lab_City
+    csvFields:
+    - name: Lab_City
 
   - name: standard.testing_lab_state
-    csvField: Lab_State
+    csvFields:
+    - name: Lab_State
 
   - name: standard.testing_lab_zip_code
-    csvField: Lab_Zip
+    csvFields:
+    - name: Lab_Zip
 
   - name: standard.testing_lab_phone_number
-    csvField: Lab_Phone
+    csvFields:
+    - name: Lab_Phone
 
   - name: standard.patient_id
-    csvField: Patient_ID
+    csvFields:
+    - name: Patient_ID
 
   - name: standard.patient_first_name
-    csvField: Patient_First_Name
+    csvFields:
+    - name: Patient_First_Name
 
   - name: standard.patient_last_name
-    csvField: Patient_Last_Name
+    csvFields:
+    - name: Patient_Last_Name
 
   - name: standard.patient_dob
-    csvField: Patient_Date_of_Birth
+    csvFields:
+    - name: Patient_Date_of_Birth
 
   - name: standard.patient_gender
-    csvField: Patient_Sex
-
-  - name: standard.patient_race#text
-    csvField: Race
+    csvFields:
+    - name: Patient_Sex
+
+  - name: standard.patient_race
```

Comment:
By putting this mapping in the AZ schema, this assumes its a unicorn.    I suspect we'l need to support many many alternative display-value-sets - we may want to revisit this, and gather them in one place.